packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -1545,10 +1545,7 @@ const Account = Backbone.Model.extend(
     async consumeRecoveryCode(code) {
       const response = await this._fxaClient.consumeRecoveryCode(
         this.get('sessionToken'),
-        code,
-        {
```

Comment:
This cropped up in test. There's a direct map her from `_fxaClient.consumRecoveryCode` to the authClient's `consumeRecoveryCode` function, which prior to this PR only took 2 arguments. That means this metrics context was never being used...

Now that I've added headers an optional 3rd param, this was getting sent into headers and was failing. Since it had no effect previously, and this code is quite old, it seems like a mistake and probably fine to simply remove the argument. I didn't notice this thing elsewhere, and tests are passing, but it does make one wonder... and I could use a sanity check here.